challenge-04/js/script.js

```diff
@@ -1,15 +1,22 @@
 var buttons = document.getElementsByTagName('button');
 var images = document.getElementsByTagName('img');
 
```

Comment:
👍The functionaries can be split into smaller function, e.g selectActiveImages, disableButton, ScollTo ....